js/lib.deno.d.ts

```diff
@@ -27,10 +27,15 @@ declare class Console {
 
 interface Window {
   console: Console;
-  mainSource: string; // TODO(ry) This shouldn't be global.
+  // TODO(ry) These shouldn't be global.
+  mainSource: string;
+  setMainSourceMap(sm: string): void;
 }
 
 // Globals in the runtime environment
 declare let console: Console;
-declare let mainSource: string; // TODO(ry) This shouldn't be global.
 declare const window: Window;
+
+// TODO(ry) These shouldn't be global.
+declare let mainSource: string;
```

Comment:
This seems duplicative with the Window interface above. Do we really need both?

Reply:
I think we don't need either - but I'm not sure the right solution now. I will fix it later.